src/managers/viewManager.ts

```diff
@@ -53,22 +33,58 @@ module powerbi.extensibility.visual {
          * @param selectionManager 
          */
         renderCalendar(options: VisualUpdateOptions, viewModel: CalendarViewModel) {
+            this.calendarSVG.remove();
             this.viewModel = viewModel;
             this.options = options;
             let currentZoomLevel = this.stateManager.getZoomLevel();
+            debugger;
             this.layoutConfig = new LayoutConfiguration(options, viewModel, currentZoomLevel);
-            if (viewModel.drillDownInfo.isDrillDown) {
-                this.yearViewLayout = new YearViewLayoutConfiguration(options.viewport.width, options.viewport.height, viewModel.configurations, viewModel.drillDownDataPoints.length);
-                this.renderDrillDownView();
+            if (this.viewModel.isLandingPage) {
+                if (!this.isLandingPageOn || options.dataViews[0].metadata.columns.length >= 3) {
```

Comment:
What does this 3 represent? Why can we not use the suggested '!options.dataViews || !options.dataViews.length'

Reply:
I made some adjustments so that the landing page will still display when only a single data field has been added. The suggested check will only show the landing page when no data fields are present. This is one of the adjustments I had to make. This one specifically makes sure the landing page remains when you switch from a regular date view to a hierarchy view and one data field is present.

Reply:
Ok, I see. In that case we will keep for now. Microsoft AppSource may see this as a potential issue, but we will keep for now. This is new functionality so not sure what expected behavior is.